source/linux/Foundational_Components/System_Security/Auth_boot.rst

```diff
@@ -0,0 +1,203 @@
+.. _auth_boot_guide:
+
+#############################
+Authenticated Boot User Guide
+#############################
+
+Introduction
+************
+
+As we head into a new world of security requirements and regulations, secure boot is the first and most essential step. Secure boot is a process that ensures only authenticated software is selected and loaded, protecting systems from unauthorized or malicious code trying to load a unknown bootloader or OS on your device.
+
+Secure boot is achieved by verifying digital signatures of each software layer involved during boot before executing that code. This requires that the design of hardware and software is prepared and developed with security in mind.
+
+Learning
+********
+
+Root of Trust (RoT)
+===================
+
+The Root of Trust is the foundation of authenticated boot. It is the first component in the system that is inherently trusted and is responsible for verifying all subsequent components in the boot process. The RoT is usually implemented in hardware, firmware, or a combination of both.
+
+There are two main types of RoT:
+
+   - *Hardware Root of Trust*: Typically embedded in a secure element (such as a Trusted Platform Module [TPM], Hardware Security Module [HSM], or Secure Boot ROM). It is immutable and performs the first-stage verification.
+
+   - *Firmware/Software Root of Trust*: This is the first code that runs on the system, typically stored in Read-Only Memory (ROM) or write-protected storage.
+
+Chain of Trust (CoT)
+====================
+
+The Chain of Trust extends the RoT by ensuring that every stage of the boot process verifies the next stage before executing it. Each stage is cryptographically signed, and verification is performed using public key cryptography.
+
+Process:
+
+   1. Boot ROM → Verifies the Primary Bootloader using a cryptographic signature.
```

Comment:
Please use standard hyphens to prevent rendering issues in other document formats